src/components/UI/DataTable.tsx

```diff
@@ -71,11 +79,26 @@ export const DataTable: FC<Props> = ({ columnHeaders, data }) => {
 
           {dataType === 'Releases' &&
             data.map((r: ReleaseData, idx: number) => {
+              const url = r?.release?.url;
+              const os = getOS(url);
+
+              const _isLinuxPrimaryRelease = isLinuxPrimaryRelease(r, os, data);
+              const _isDarwinPrimaryRelease = isDarwinPrimaryRelease(r, os, data);
+              const _isWindowsPrimaryRelease = isWindowsPrimaryRelease(r, os, data);
+              const _isMobilePrimaryRelease = isMobilePrimaryRelease(r, os, data);
+
+              const isPrimaryRelease =
+                _isLinuxPrimaryRelease ||
+                _isDarwinPrimaryRelease ||
+                _isWindowsPrimaryRelease ||
+                _isMobilePrimaryRelease;
+
               return (
                 <Tr
                   key={idx}
                   transition={'all 0.5s'}
                   _hover={{ background: 'button-bg', transition: 'all 0.5s' }}
+                  fontWeight={isPrimaryRelease ? 700 : 400}
```

Comment:
@nloureiro This is fairly noticeable on light mode:
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/54227730/207197975-cc7dd755-b9f2-43f0-9147-d25c4912eb76.png">
But on dark mode is a bit difficult to distinguish:
<img width="1504" alt="image" src="https://user-images.githubusercontent.com/54227730/207198059-2aba8829-fb48-4646-83de-c6a27853952d.png">

Just curious your thoughts on maybe increasing the font weight even more, or adding some other styling to this to make it more identifiable.

Reply:
@wackerow I wasn't importing the font correctly, now 700 is more noticeable on light/dark